deriv.com.conf

```diff
@@ -102,4 +102,16 @@ server {
     location ~* \.(pdf)$ {
         expires 5m;
     }
+    pagespeed FileCachePath /var/ngx_pagespeed_cache;
+
+    location ~ "\.pagespeed\.([a-z]\.)?[a-z]{2}\.[^.]{10}\.[^.]+" {
+     add_header "" "";
+    }
+    location ~ "^/pagespeed_static/" { }
+    location ~ "^/ngx_pagespeed_beacon$" { }
+
+    location = /50x.html {
+       root   html;
+    }
```

Comment:
better to avoid this block as we have already declared error_page to 50x status codes